content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -73,7 +83,9 @@ The configuration of the module explains how to configure filters. Filters are u
 Each user notification has a list of recipients. The recipient is an entity in backstage catalog. The notification will be sent to the email addresses of the recipients.
 
 ## Broadcast notifications
-In broadcast notifications we do not have recipients. The module's configuration supports a few options for broadcast notifications:
- - Ignoring broadcast notifications
- - Sending to predefined address list
- - Sending to all users
+In broadcast notifications we do not have recipients, the notifications are delivered to all users.
+
+The module's configuration supports a few options for broadcast notifications:
+ - Ignoring broadcast notifications to be forwarded
```

Comment:
I might add (this will still send a notification to the user that executes the workflow if the workflow user has an email configured)

Reply:
`A workflow` is unrelated in this context, a workflow is just one of the using/consuming parties, notifications are more generic feature.

In the context of these two plugins only, if a notification is created (means a single POST request), it is either of the entity or broadcast type - not combined.

For entity-typed, just the recepients are notified via email.
For broadcast, there is no explicit recipient and the boradcastConfig is used instead.

Your comment is most probably refering to one of the recent issues but this has been (hopefuly) resolved (PR merged yesterday)